cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -286,6 +281,29 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 	return cvc, nil
 }
 
+func (c *CVCController) getVolumePolicy(
+	policyName string,
+	cvc *apis.CStorVolumeClaim,
+) (*apis.CStorVolumePolicy, error) {
+
+	volumePolicy := &apis.CStorVolumePolicy{}
+	var err error
+
+	if policyName != "" {
```

Comment:
nit: Instead of handling over here we can handle in caller code itself.

Reply:
Both are same, i just moved below to make it more cleaner